pagefaultgames/pokerogue
# [Bug] Fix Animation for Variant Exp Meadow Vivillon

## What are the changes?
Rare and epic shiny Meadow Pattern Vivillon (n666)'s Experimental animations are now appropriate for its spritesheet, instead of taken from the non-shiny spritesheet, and which uses a completely different sprite.

## Why am I doing these changes?
It's been reported! A lot!
https://discord.com/channels/1125469663833370665/1238339524778790922/1250180831469899808
https://discord.com/channels/1125469663833370665/1238339524778790922/1250018721729220668
https://discord.com/channels/1125469663833370665/1238339524778790922/1250018721729220668
And yeah, ideally the animation a Pokemon gets is the right one for its sprite.
![image](https://github.com/pagefaultgames/pokerogue/assets/31082757/c59f9c0e-2bf1-47a9-b403-22c930c8189b)

## What did change?
Took the animation frame data from Fancy Vivillon (rare), which animates properly and has an identically shaped spritesheet.
This does mean the Texture Packer link is now Not Correct, and a new one was not generated in its place. If someone still regularly uses TP lmk.

### Screenshots/Videos
#### Before
![image](https://github.com/pagefaultgames/pokerogue/assets/31082757/d3b1f493-5001-4c36-be8e-3a3233226010)

#### After
![image](https://github.com/pagefaultgames/pokerogue/assets/31082757/fdd670a7-d7f8-4d15-9244-a7beb157391f)
![image](https://github.com/pagefaultgames/pokerogue/assets/31082757/05b50ff3-e65d-40e0-b5ed-f6e930172b99)


## How to test the changes?
Overrides were used. Meadow Pattern Vivillon are relatively common in the first several areas, so `OPP_SPECIES_OVERRIDE` does work, but one could also simply bring in a variant Meadow Pattern Spewpa (n664), possibly with its starting level increased. If it is possible to override form at the same time as species please do let me know.
Ensuring that the `Sprite Set` used is `Mixed Animated` will be necessary to see the changes.

## Checklist
